communications/slack.md

```diff
@@ -0,0 +1,91 @@
+## On Slack
+
+Slack is a resourceful tool being used by a complex company. Each team will have specific dynamics that are from the scope of their tasks and it might require diverse communication tools.
+
+This is a gathering of some of the most used resources in slack to efficiently communicate based on takes from different people within the company. 
+
+Have in mind who you're trying to reach: If the person you want to correspond with is a highly contested resource, the expectation from them to engage with you synchronously will be frustrating. That's why we say `default to async communication`. 
+
+**Opt-in**: Join channels that you are interested in or obligated to.
+
+**Minimize Distractions**: Nowadays we have way too many channels of receiving information, by choice or otherwise. Instead of striving to achieve "Notification Zero", try to be as close as possible to "Everything Zero". This means diligently clear or defer any information that is not work-related. 
+
+Custom display channel: Star the channel heavily used, and only display unread channels if you prefer a clean interface. Click with the left button on the channel and select `Star channel`. 
+<p align="center">
+ <img width="200" alt="star" src=https://user-images.githubusercontent.com/53827690/89816335-63e4f680-db14-11ea-85dc-b42ce633a89a.png>
+ <img width="400" alt="display" src=https://user-images.githubusercontent.com/53827690/89817201-becb1d80-db15-11ea-9057-3fbd7ac038c0.png>
+</p>
+
+Use the sidebar section to organize channels in different use cases. For example, all dev related channels can be under a `Team` section to remind of the highest priority to read.
+
+<p align="center">
+ <img width="200" alt="section" src=https://user-images.githubusercontent.com/53827690/89816591-d8b83080-db14-11ea-931b-21b8b00934c5.png>
+ <img width="400" alt="section-choice" src=https://user-images.githubusercontent.com/53827690/89816656-eec5f100-db14-11ea-96f0-7c3514829e4f.png>
+</p>
+
+Some platforms like Slack offer one simple toggle to enter the "Do Not Disturb" mode. Use it to minimize distractions, so you can find what you need to focus on, faster.
+<p align="center">
+ <img width="400" alt="do not disturb" src=https://user-images.githubusercontent.com/53827690/89809928-46ab2a80-db0a-11ea-8e05-324750c2c748.png>
+ </p>
+
+**Notifications**: Once you're part of a channel, set up your notifications to suit your needs:
```

Comment:
I feel Mentions $ reactions is a nice feature to introduce as well. No need to go to each channel, one place to see all messages that related or mentioned you.
![image](https://user-images.githubusercontent.com/20359409/89828113-9a2b7180-db26-11ea-9128-433f6bf0377b.png)